src/Stratis.Bitcoin.Tests/Consensus/ChainedHeaderTreeTest.cs

```diff
@@ -52,7 +53,8 @@ internal TestContextBuilder WithInitialChain(int initialChainSize, bool assignBl
                 if (initialChainSize < 0)
                     throw new ArgumentOutOfRangeException(nameof(initialChainSize), "Size cannot be less than 0.");
 
-                this.testContext.InitialChainTip = this.testContext.ExtendAChain(initialChainSize, assignBlocks: assignBlocks);
+                this.testContext.InitialChainTip =
```

Comment:
Could you please configure your visual studio so that you do not enforce such strict limits on the number of characters in the line?